lib/charliecloud.py

```diff
@@ -50,6 +51,24 @@ class Force_Mode(enum.Enum):
    SECCOMP="seccomp"
    NONE="none"
 
+# Log level
+@functools.total_ordering
+class Log_Level(enum.Enum):
+   TRACE = 3
+   DEBUG = 2
+   VERBOSE = 1
+   INFO = 0
+   QUIET_INFO = -1
+   QUIET_WARNING = -2
+   QUIET_STDERR = -3
```

Comment:
More idiomatic would be something like:

```suggestion
   WARNING = -1
   FATAL = -2
   FATAL_SILENCE = -3
```